checks/prgenv/helloworld.py

```diff
@@ -214,39 +153,40 @@ def set_omp_env_variable(self):
 
 @rfm.simple_test
 class HelloWorldTestMPI(HelloWorldBaseTest):
-    sourcesdir = 'src/mpi'
     # for the MPI test the self.num_tasks_per_node should always be one. If
```

Comment:
is this still valid ?

Reply:
I say we leave it to make sure that it's consistent. I will set the num nodes and remove the comment if you want